.argo-ci/ci.yaml

```diff
@@ -0,0 +1,137 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Workflow
+metadata:
+  generateName: argo-ci-
+spec:
+  entrypoint: argo-ci
+  arguments:
+    parameters:
+    - name: revision
+      value: master
+    - name: repo
+      value: https://github.com/argoproj/argo.git
+
+  templates:
+  - name: argo-ci
+    inputs:
+      parameters:
+      - name: revision
+      - name: repo
+    steps:
+    - - name: controller-image
+        template: ci-builder
+        arguments:
+          parameters:
+          - name: revision
+            value: "{{inputs.parameters.revision}}"
```

Comment:
Rather than pass these values around, lets use the new global parameter feature. These are now referencable using `{{workflow.parameters.revision}}`

Reply:
Thanks for pointing this out. Fixed: https://github.com/argoproj/argo/pull/619/commits/1d4f14dad682966ebc08cdcb8837913aeb556a24